app/uk/gov/hmrc/individualsmatchingapi/controllers/CommonController.scala

```diff
@@ -83,7 +83,7 @@ abstract class CommonController @Inject()(cc: ControllerComponents)(implicit exe
       ErrorInvalidRequest(e.getMessage).toHttpResponse
   }
 
-  private[controllers] def recoveryWithAudit(correlationId: Option[String], matchId: String, url: String)(
+  private[controllers] def recoveryWithLogging(correlationId: Option[String], matchId: String, url: String)(
```

Comment:
Actually, I do not see any logging in here, it is only sending audit events. So, I do not believe we need to change the function and we do not need to redact any content.